scenarioo-docu-generation-example/src/test/java/org/scenarioo/uitest/example/infrastructure/UseCaseDocuWritingRule.java

```diff
@@ -81,7 +81,7 @@ public void evaluate() throws Throwable {
 	public static UseCase createUseCase(final Class<?> testClass) {
 		// Extract usecase name and description from concrete test class.
 		String description = "";
-		String name = createUseCaseName(testClass);
+		String name = UseCaseConfiguration.getUseCaseName(testClass, MultipleBuildsRule.getCurrentBuildRun());
```

Comment:
Same as for Scenario Name generation, see other comment. "UseCaseConfiguration" is not a good name here and it should only do the special dummy name modifications for those builds where there have to be used different names.